source/Ship.cpp

```diff
@@ -2515,6 +2545,13 @@ double Ship::DisruptionLevel() const
 
 
 
+bool Ship::IsCloakTargetable() const
```

Comment:
Why are you using this instead of `Ship::IsTargetable`? Pretty sure all usages of this function can be replaced by either `IsTargetable` or `IsCloaked`.

Reply:
It is !IsCloaked by definition, but this is much clearer? I can change it yeah
it just checks cloaking < 1 ~~also this way we can change how cloak targetability works~~